blender/arm/material/make_voxel.py

```diff
@@ -1,29 +1,251 @@
 import bpy
-
 import arm.utils
 import arm.assets as assets
+import arm.material.cycles as cycles
 import arm.material.mat_state as mat_state
-
-if arm.is_reload(__name__):
-    arm.utils = arm.reload_module(arm.utils)
-    assets = arm.reload_module(assets)
-    mat_state = arm.reload_module(mat_state)
-else:
-    arm.enable_reload(__name__)

```

Comment:
Looks like the reloading code was removed by accident?

Reply:
I bet same thing that change above, when I rebased I had to skip some commits related to voxels (they were named explicitly got zeit dank), but most probably I just mingled when doing very repetitive actions.